Formula/lcov.rb

```diff
@@ -1,6 +1,6 @@
 class Lcov < Formula
   desc "Graphical front-end for GCC's coverage testing tool (gcov)"
-  homepage "http://ltp.sourceforge.net/coverage/lcov.php"
+  homepage "https://ltp.sourceforge.io/"
```

Comment:
This just directs the user to GitHub, so let's switch to https://github.com/linux-test-project/lcov as the homepage.